astropy/coordinates/angles/_wrap.pyx

```diff
@@ -0,0 +1,55 @@
+#cython: language_level=3
+cimport cython
+cimport numpy as np
+from libc cimport math
+
+np.import_array()
+
+ctypedef fused dtype:
```

Comment:
`dtype` feels unexpected here since it has a different meaning in numpy, so I'd suggest using something more idiomatic in templates, like `T`
```suggestion
ctypedef fused T:
```

Reply:
well the file you linked uses it to fuse numpy data types, which in my view is a little different. But in any case this is just a nit. If nobody else finds this confusing feel free to ignore this suggestion.